docs/products/clickhouse/howto/query-databases.md

```diff
@@ -15,6 +15,13 @@ the entire cluster. In the standard ClickHouse, the queries `CREATE`,
 In contrast, we ensure the proper distribution across all cluster
 machines behind the scenes. You don't need to remember using
 `ON CLUSTER` for every query.
+
+:::important
+There are limits on the number of queries that you can run concurrently in Aiven for
```

Comment:
"There are limits on the number of concurrent connections & queries you can run in Aiven for ..." as now this is covering both!